### PR TITLE
[android][ios] Upgrade react-native-pager-view to 6.1.2

### PR DESCRIPTION
**Pull request by gabrieldonadel**
# Why

Closes ENG-7320

# How
 
```sh
et uvm -m react-native-pager-view -c v6.1.2
et pods -f
yarn 
```

# Test Plan
 
- [x] test unversioned expo go ios + NCL Pager View
- [x] test unversioned expo go android + NCL Pager View

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: apps/bare-expo/ios/Podfile.lock [hunk 1]
```diff
@@ -548,9 +548,9 @@ PODS:
   - React-jsinspector (0.71.0)
   - React-logger (0.71.0):
     - glog
-  - react-native-netinfo (9.3.5):
+  - react-native-netinfo (9.3.7):
```
Comment (gabrieldonadel) [inline apps/bare-expo/ios/Podfile.lock:551]: leftover of https://github.com/expo/expo/commit/e4d7e21f0c15d2488232bcd941e5619830f1b7e5

File: android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/pagerview/PagerViewViewManager.kt [hunk 2]
```diff
@@ -1,281 +0,0 @@
-package versioned.host.exp.exponent.modules.api.components.pagerview
```
Comment (tsapeta) [inline android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/pagerview/PagerViewViewManager.kt:1]: Looks like there are many changes in this and the new `PagerViewViewManagerImpl`, so I just want to be sure — have you tested that in Expo Go?

File: packages/html-elements/package.json [hunk 3]
```diff
@@ -38,15 +38,21 @@
     "projects": [
       {
         "preset": "jest-expo/ios",
-        "modulePathIgnorePatterns": ["<rootDir>/babel/"]
+        "modulePathIgnorePatterns": [
```
Comment (tsapeta) [inline packages/html-elements/package.json:41]: See my comment from Doug's PR: https://github.com/expo/expo/pull/20913#discussion_r1083429167
I'm wondering why is this even trying to change this file 🤔

**Review by tsapeta — APPROVED** [review]

File: packages/html-elements/package.json (hunk 3 above)
Comment (gabrieldonadel) [reply packages/html-elements/package.json:41]: hmm, I also see that on https://github.com/expo/expo/pull/20933, perhaps it has something to do with the `et uvm` command? Let me revert this

Comment (gabrieldonadel) [reply packages/html-elements/package.json]: fixed by rebasing this with `main`

**Review by tsapeta — APPROVED** [review]

File: android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/pagerview/PagerViewViewManager.kt (hunk 2 above)
Comment (gabrieldonadel) [reply android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/pagerview/PagerViewViewManager.kt:1]: I wasn't able to build Expo Go yesterday due to an unrelated issue so I ended up not noticing this, but indeed this isn't working on Expo Go, it's erroring out about an `Unresolved reference: PagerViewViewManager`

**Comment by gabrieldonadel** [comment]
Marking this back as a draft while I fix the Paper/Fabric folders for PagerViewViewManager

File: android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/pagerview/PagerViewViewManager.kt (hunk 2 above)
Comment (gabrieldonadel) [reply android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/pagerview/PagerViewViewManager.kt:1]: I had to migrate `react-native-pager-view` to `expoGoConfig` but everything seems to be working fine now

**Review by tsapeta — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,6 +14,7 @@ Package-specific changes not released in any SDK will be added here just before
 - Updated `@shopify/react-native-skia` from `0.1.157` to `0.1.171`. ([#20857](https://github.com/expo/expo/pull/20857) by [@kudo](https://github.com/kudo))
 - Updated `react-native-safe-area-context` from `4.4.1` to `4.5.0`. ([#20899](https://github.com/expo/expo/pull/20899) by [@gabrieldonadel](https://github.com/gabrieldonadel))
 - Updated `react-native-screens` from `3.18.0` to `3.19.0`. ([#20938](https://github.com/expo/expo/pull/20938) by [@lukmccall](https://github.com/lukmccall))
+- Updated `@react-native-community/slider` from `6.0.1` to `6.1.2`. ([#20932](https://github.com/expo/expo/pull/20932) by [@gabrieldonadel](https://github.com/gabrieldonadel))
```
Comment (buraks) [inline CHANGELOG.md:17]: I believe there is a typo here, should have been `react-native-pager-view`

Comment (gabrieldonadel) [reply CHANGELOG.md:17]: Thanks for highlighting this @buraks, this has been fixed on https://github.com/expo/expo/commit/5cb3fbeaa9400947e854ff31ee078859347f7bbd

**Pull request merged**